### DISCUSSION TITLE
thumbnail for pdfs issues

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#636] [Feature Requests]
I uploaded a pdf. Since there is no pdf preview yet, the pdf had a boring placeholder thumbnail. So I added an image to the pdf to serve as its thumbnail. However, the thumbnail for the pdf didn’t update. 

On top of that, the category I had assigned to the pdf now has the image as its thumbnail. Even if I add more media to the category, the category thumbnail does not update, it keeps displaying the image. 

And on top of that, checking ‘clear’ next to the image and updating the pdf does not clear the image. The image is forever.

Please fix thumbnails for pdfs.

**Comment by whatnik** [comment]
It looks like the error is related to incorrect file paths, by example http://video.itimho.ru/media/home/mediacms.io/mediacms/media_files/original/thumbnails/user/supertest/logo_dark.jpg (bad)
http://video.itimho.ru/media/original/thumbnails/user/supertest/logo_dark.jpg (work)
